docs/enterprise/concepts.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Concepts
+sidebarDepth: 1
+description: Learn how the Pomerium Enterprise Console works.
+---
+
+# Concepts
+
+## Namespaces
+
+In the Pomerium Enterprise Console, a namespace is where you can define an organizational unit of users and groups with fine-grained access management. This enables teams to self-service the routes and policies pertinent to them. Namespaces can optionally inherit from their parent units.
+
+## Service Accounts
+
+Service accounts handle machine-to-machine communication from Pomerium to your Identity Provider (**IdP**) in order to retrieve and establish group membership. Configuration is largely dependent on the IdP, but is usually an API acccess token with sufficient privlidges to read users and groups.
```

Comment:
Service accounts in the console context are to facilitate machine-to-machine _through_ the pomerium proxy.  Most concrete use case is interacting with the console API but other programmatic access to systems behind the proxy are supported.  

Examples would be monitoring systems, API integrations, etc.